website/docs/d/org_vdc.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org_vdc"
+sidebar_current: "docs-vcd-data-source-org-vdc"
+description: |-
+  Provides a organization VDC data source.
+---
+
+# vcd\_org\_vcd
+
+Provides a vCloud Director Organization VDC data source. A Organization VDC can be used to reference a VCD and use its 
```

Comment:
Just as above. "an organization VDC can be used"

Reply:
fixed